config/cscs.py

```diff
@@ -768,19 +768,17 @@
                         },
                     ],
                     'launcher': 'srun',
-                    'processor': [
-                        {
-                            'arch': 'AMD EPYC 7742 64-Core Processor',
-                            'num_cpus': 256,
-                            'num_cpus_per_core': 2,
-                            'num_cpus_per_socket' 128,
-                            # +num_cores
-                            # +num_cores_per_socket
-                            # +num_numa_nodes
-                            # +num_cores_per_numa_node
-                            # 'num_sockets'
-                        },
-                    ],
+                    'processor': {
+                        'arch': 'AMD EPYC 7742 64-Core Processor',
```

Comment:
This should be compatible with [archspec](https://archspec.readthedocs.io/en/latest/index.html). After installing you can see the name with
```
python -c "import archspec.cpu; print(archspec.cpu.host())"
```

Reply:
I think it should be `zen2` in pilatus.
```suggestion
                        'arch': 'zen2',
```